templates/contribute/naming.md

```diff
@@ -144,13 +144,14 @@ The symbols `≤` and `<` have a special naming convention.
 In mathlib, we almost always use `≤` and `<` instead of `≥` and `>`, so we can use both `le`/`lt` and `ge`/`gt` for naming `≤` and `<`.
 There are a few reasons to use `ge`/`gt`:
 
-1. We use `ge`/`gt` if the arguments to `≤` or `<` appear in different orders. Then `ge`/`gt` indicates that the arguments are swapped.
+1. We use `ge`/`gt` if the arguments to `≤` or `<` appear in different orders. We use `le`/`lt` for the first occurrence of `≤`/`<` in the theorem name, and then `ge`/`gt` indicates that the arguments are swapped.
```

Comment:
Good practice to line-wrap reasonably often, eg at punctuation. this helps reduce the frequency of git conflicts
```suggestion
1. We use `ge`/`gt` if the arguments to `≤` or `<` appear in different orders.
  We use `le`/`lt` for the first occurrence of `≤`/`<` in the theorem name,
  and then `ge`/`gt` indicates that the arguments are swapped.
```